### PR TITLE
Liquid assets polish

### PR DESCRIPTION
**Pull request by fcduarte**
- Transactions list refinements
- Add a border on Summary view
- Initial polish of the charts

/cc @amitbharadwaj @josemontesdeoca

### CODE REVIEW

**Comment by josemontesdeoca** [comment]
Overall good. Few thing to improve:
- Transaction list is not showing Debits transactions (the ones that increase the total cash)
- ActionBar should read "Transactions"
- Label "LIQUID ASSETS" should read "TOTAL CASH"
- background should be white for where the graph and overview data is.

File: res/drawable/left_border.xml [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
-    <item>
+    <item android:id="@+id/left_border_shape">
         <shape android:shape="rectangle" >
             <solid android:color="@color/navy_blue" />
         </shape>
     </item>
-    <item android:bottom="3dp">
+    <item android:left="3dp">
         <shape android:shape="rectangle" >
             <solid android:color="@color/lightgray" />
```
Comment (josemontesdeoca) [inline res/drawable/left_border.xml:11]: From the specs: https://github.com/CodePath-MAF/SharedAssets/blob/master/Android/_specs/02_a_Transactions.png

this color should be #F3F3F3

**Comment by josemontesdeoca** [comment]
Also, can the expandable list be open in its initial behavior?

**Comment by fcduarte** [comment]
@josemontesdeoca you mean all open? If so there's only a ugly hack .. Iterate over the groups and call `expand()`

**Comment by fcduarte** [comment]
Changes will be on the next PR. Merging...

**Comment by josemontesdeoca** [comment]
@fcduarte yes I meant all open.

**Pull request merged**